ql/termstructures/yield/oisratehelper.cpp

```diff
@@ -198,7 +201,14 @@ namespace QuantLib {
         QL_REQUIRE(termStructure_ != nullptr, "term structure not set");
         // we didn't register as observers - force calculation
         swap_->deepUpdate();
-        return swap_->fairRate();
+        // weak implementation... to be improved
```

Comment:
I'm not sure what I meant with this.  Maybe that we're doing the calculation here instead of calling some swap methods?  But then again, as you've seen, we can't build the swap with the correct spread since the value in the handle might change.  I don't know.  You can probably remove or change the comment.